tokenExchange/src/app/tokenGen.js

```diff
@@ -5,10 +5,10 @@ const base64url = require("base64url");
 
 module.exports = {
     async generateToken(decodedToken){
-        const keyId = process.env.KEY_ID;
```

Comment:
E' possibile usare l'alias per recuperare la chiave, ma poi va messo il "kid" nel header del token jwt e il toke fa firmato con la stessa kid (usando l'alias potrebbe essere modificata la chiave tra il momento che viene letto il kid e messo nel token e il momento in cui viene firmato il token stesso). L'importante che ci sia coerenza tra kid scritto nel token e quella usata per la firma.